public/axioms.html

```diff
@@ -11,21 +11,23 @@
   </head>
   <body>
     <header>
-      <h1>axioms</h1>
-      <a href="index.html" align="center"><p>←back to home</p></a>
-      <p align="center" class="pageDescription">
-        "in fact, every insurrectionary process starts from a truth that we
-        refuse to give up."
-      </p>
+      <div class="pagename">
+        <h1>axioms</h1>
+        <a href="index.html"><p>←back to home</p></a>
+        <p class="pageDescription">
+          "in fact, every insurrectionary process starts from a truth that we
+          refuse to give up."
+        </p>
+    </div>
     </header>
 
     <div id="axioms" align="left">
       <br />
       <ol class="indentList" type="i">
         <li>
-          <h2 class="listTitle">
+          <h3 class="listTitle">
```

Comment:
Heading elements should always follow a descending order from h1 to h2 to h3 etc. All font-size and style changes should be made with the use of CSS and specific classes.